hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2033,7 +2024,11 @@
       name: chip_sw_csrng_lc_hw_debug_en
       desc: '''Verify the effect of LC HW debug enable on CSRNG.
 
-            TODO: This is pending SCA security review and might be removed.
+            lc_hw_debug_en is used to diversify the csrng seed.
+            - Instantiate CSRNG with fixed seed and fetch entropy while in TEST state (hw_debug_en = 1).
+            - Save the retrieved entropy in flash and advance the device to PROD or DEV state.
+            - Instantiate CSRNG with the same fixed seed and fetch entropy again while in the NEW state.
```

Comment:
Unless there is a mode or trick being done here, doubt that this sequence is possible.

Reply:
i think maybe i misunderstood what flag0 does, i thought it meant we could supply with a fixed value.